### ISSUE TITLE
Autocompletion/Diagnostics broken when accessing property on object within moustache tag (`bla.`)

### ISSUE THREAD

**Issue by lake2** [sveltejs/language-tools#538]
code：
```js
{#if user.loggedIn}<button on:click={toggle}> Log out </button>{/if}

<script>
    let user = { loggedIn: false };

    function toggle() {
        user.loggedIn = !user.loggedIn;
    }
</script>
```
when type `user.`，auto complete is broken：

![image](https://user-images.githubusercontent.com/6293752/93059583-7fa76380-f6a3-11ea-94b8-be4d61512143.png)

when type `user.a` this is no error:
```js
// expect an error here 
{#if user.a}<button on:click={toggle}> Log out </button>{/if}

<script>
    let user = { loggedIn: false };

    function toggle() {
        user.loggedIn = !user.loggedIn;
    }
</script>

```

**Comment by dummdidumm** [comment]
Blocked by https://github.com/sveltejs/svelte/issues/4818

**Comment by jasonlyu123** [comment]
Maybe we can replace something like `a.}` and `a?.}` to `{a }`  `{a  }` before we hand it over to the svelte compiler. That won't fix all the issue but can workaround simple one. But is it worth the hassle and will it introduce some unwanted behavior?

**Comment by dummdidumm** [comment]
I think that's a good idea to explore. I guess if it sometimes will result in strange behavior (maybe, don't know) it will still be better than the current behavior.

**Comment by dummdidumm** [comment]
The mentioned use case of "`.` before `}` is fixed. There surely are other cases (for example #776) where intellisense will not work properly. For these, please open a separate issue.
